common/changes/@itwin/core-frontend/daniel-BENTLEY_materials_line_style_2025-12-12-00-01.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Consider adding a NextVersion.md entry.  See https://github.com/iTwin/itwinjs-core/blob/765100c026dcb952ac005620a609001b6bee05ce/docs/changehistory/NextVersion.md for the BENTLEY_materials_point_style entry.